core/arch/arm/include/kernel/cache_helpers.h

```diff
@@ -31,6 +32,24 @@ void dcache_op_all(unsigned long op_type);
 void dcache_op_level1(unsigned long op_type);
 void dcache_op_level2(unsigned long op_type);
 void dcache_op_level3(unsigned long op_type);
+
+/*
+ * Get system cache line size from ARM system configuration registers.
+ */
+static inline uint32_t dcache_get_line_size(void)
```

Comment:
There's no need for a fixed width integer here, please change to `unsigned int`

Reply:
Ol